op-service/txmgr/metrics/tx_metrics.go

```diff
@@ -26,9 +49,72 @@ func MakeTxMetrics(ns string, factory metrics.Factory) TxMetrics {
 			Help:      "L1 gas fee for transactions in GWEI",
 			Subsystem: "txmgr",
 		}),
+		currentNonce: factory.NewGauge(prometheus.GaugeOpts{
+			Namespace: ns,
+			Name:      "current_nonce",
+			Help:      "",
```

Comment:
No help text on all the shiny new metrics?

Reply:
not yet. I try to have the names be descriptive enough